### PR TITLE
Docs: add `Logging.BelowMinLevel` and `Logging.AboveMaxLevel` to the manual page for the Logging stdlib

### PR DESCRIPTION
**Pull request by DilumAluthge**
According to the source code file, `Logging.BelowMinLevel` and `Logging.AboveMaxLevel` are already part of the public API of the Logging stdlib:

https://github.com/JuliaLang/julia/blob/c8ca350832030992fca113ed56d979b6a8ff7fd3/stdlib/Logging/src/Logging.jl#L85-L91

Additionally, Logging currently exports `BelowMinLevel` and `AboveMaxLevel`:

https://github.com/JuliaLang/julia/blob/c8ca350832030992fca113ed56d979b6a8ff7fd3/stdlib/Logging/src/Logging.jl#L61-L81

However, `Logging.BelowMinLevel` and `Logging.AboveMaxLevel` are not currently in the manual.

This PR adds them to the manual.

### CODE REVIEW

File: stdlib/Logging/src/Logging.jl [hunk 1]
```diff
@@ -54,6 +54,18 @@ const Warn = Base.CoreLogging.Warn
 Alias for [`LogLevel(2000)`](@ref LogLevel).
 """
 const Error = Base.CoreLogging.Error
+"""
+    BelowMinLevel
+
+Alias for [`LogLevel(-1_000_001)`](@ref LogLevel).
```
Comment (ericphanson) [inline stdlib/Logging/src/Logging.jl:60]: imo it would be helpful to have a more descriptive docstring; e.g. what are these for? how should they be used?

Comment (DilumAluthge) [reply stdlib/Logging/src/Logging.jl:60]: Yeah, I think that would be good. I'm not sure what the intended usage is. I've been using `BelowMinLevel` in `Logging.disable_logging(Logging.BelowMinLevel)`, which I use as a workaround for #52234.

@c42f Could you weigh in on the intended usage of `BelowMinLevel` and `AboveMaxLevel`?

**Comment by DilumAluthge** [comment]
Bump @c42f

**Review by fingolfin — APPROVED** [review]
It would be great if @c42f could chime in, but IMHO even without that this is an improvement (yeah maybe it could be even better, but what can't be...)

File: stdlib/Logging/src/Logging.jl (hunk 1 above)
Comment (vtjnash) [reply stdlib/Logging/src/Logging.jl:60]: It looks like none of the others have a useful description either, so this seems to fit in

**Pull request merged**